home/models.py

```diff
@@ -79,3 +79,17 @@ class CornerBanner(models.Model):
 
     def __str__(self):
         return self.text
+
+
+@register_snippet
+class MastodonVerification(models.Model):
+    name = models.CharField(max_length=255)
+    url = models.URLField()
```

Comment:
```suggestion
    url = models.URLField(help_text='eg. fosstodon.org/@pycascades')
```

<img width="239" alt="image" src="https://user-images.githubusercontent.com/5224106/206646906-841a2d0a-accb-4fb5-984d-8a36c8031e54.png">

Reply:
I think for `URLFields` it doesn't actually matter! A scheme could be added or not, the validation will add it if it isn't present. Certainly _looks_ prettier if we add the scheme ourselves (something I do in #44), I think I'll do that.